advisor-go/src/quixo/scorer/scorer.go

```diff
@@ -0,0 +1,69 @@
+package scorer
+
+import (
+	"quixo/game"
+)
+
+/*
+ * GetGridScore return the score of the board
+ * the score of the board is the max score of all straight lines or diagonal
+ */
+func GetBoardScore(board game.Board) int {
```

Comment:
I don't understand that it doesn't depend on the player

Reply:
It's the score for the player. If 4 cubes belonging to the player are aligned, it'll return 4. The cubes of the other player don't count.